### PR TITLE
Fix broken tabs

### PR DESCRIPTION
**Pull request by mairaw**
Tabs are broken for dotnet page:

image

[Internal preview url](https://review.docs.microsoft.com/en-us/dotnet/core/tools/dotnet?branch=pr-en-us-2904&)

### CODE REVIEW

File: docs/core/tools/dotnet.md [hunk 1]
```diff
@@ -19,9 +19,13 @@ ms.technology: dotnet-cli
 ## Synopsis
 
 # [.NET Core 2.x](#tab/netcore2x)
+
 `dotnet [command] [arguments] [--additional-deps] [--additionalprobingpath] [-d|--diagnostics] [--fx-version] [-h|--help] [--info] [--roll-forward-on-no-candidate-fx] [-v|--verbose] [--version]`
+
 # [.NET Core 1.x](#tab/netcore1x)
+
 `dotnet [command] [arguments] [--additionalprobingpath] [-d|--diagnostics] [--fx-version] [-h|--help] [--info] [-v|--verbose] [--version]`
+
```
Comment (JRAlexander) [inline docs/core/tools/dotnet.md]: Still broken. Remove blank lines before each hr. Tabs don't like them.

**Review by JRAlexander — CHANGES_REQUESTED** [review]
A couple of minor changes.

File: docs/core/tools/dotnet.md (hunk 1 above)
Comment (mairaw) [inline docs/core/tools/dotnet.md]: That's how it was before, no @JRAlexander? I'll change to the code block. It works on the other topics like that.

Comment (JRAlexander) [inline docs/core/tools/dotnet.md]: ok.

**Review by JRAlexander — APPROVED** [review]

**Comment by rpetrusha** [comment]
@mairaw, I have no idea why, but although the tabs are rendering (that is, there are separate tabs for 1.x and 2.x, as expected), their content is identical.

**Comment by mairaw** [comment]
@rpetrusha it's not actually switching. I'll try one more thing to see if I can workaround this.

/cc @EisenbergEffect - one issue we're facing with tabs

**Comment by EisenbergEffect** [comment]
@jdanyow It looks like we might have a bug in tabs. Can you investigate?

@mairaw Is there a chance that I could get added as a member of the dotnet org? Notifications for dotnet are going to my personal email address instead of my Microsoft address, so I sometimes don't see these in a timely fashion. I'd love to improve that.

**Comment by mairaw** [comment]
I think I might have found the issue @EisenbergEffect @jdanyow

**Comment by EisenbergEffect** [comment]
@mairaw Was it a syntax problem or is there a legitimate bug we need to follow up on?

**Comment by mairaw** [comment]
I had a syntax problem but I'll make some more tests tomorrow to make sure @EisenbergEffect

**Comment by EisenbergEffect** [comment]
Sounds good. If you confirm the syntax issue, I'd love to understand the details of that as well. We hope to put some syntax validation in place as part of Gauntlet and OPS in the future.

**Comment by rpetrusha** [comment]
@mairaw, this looks as if it builds correctly now!

**Pull request merged**